src/es/components/molecules/openingHours/OpeningHours.js

```diff
@@ -49,15 +49,15 @@ export default class OpeningHours extends Shadow() {
         table {
             text-align: left;
             width: 100%;
-            max-width: 320px;
+            max-width: var(--opening-hours-width, 320px);
             table-layout: fixed;
             font: var(--mdx-sys-font-fix-body1);
         }
         tr {
             vertical-align: top;
         }
         .opening-hours__day-label {
-            width: 40%;
+            width: 50%;
```

Comment:
Maybe a new variable --opening-hours-day-label-width should be introduce here with 50% as default for better maintenance (can be done later).